### PR TITLE
DO NOT MERGE: DPE-3735 Add how-to guide for alert integration

### PR DESCRIPTION
**Pull request by Batalex**

### CODE REVIEW

File: docs/how-to/h-integrate-alerts.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# Integrate customer alerting rules and dashboards
+
+This guide shows you how to integrate an existing set of rules to your Charmed Kafka and Charmed Zookeeper deployment to be consumed with the [Canonical Observability Stack (COS)](https://charmhub.io/topics/canonical-observability-stack).
+To do so, we will sync the alert rules stored in a git repo to COS Lite.
+
+## Prerequisites
+
+Refer to the [Enable Monitoring](/t/charmed-kafka-documentation-how-to-enable-monitoring/10283) guide to deploy the cos-lite bundle in a Kubernetes environment and relate the Charmed Kafka model to the COS offers.
+The rest of this guide works on the assumption that two models are deployed: `cos` (on k8s) containing the cos-lite bundle and `kafka-dev` (on VM **TODO: adapt for k8s version**) containing 5 Zookeeper units and **2** Kafka.
```
Comment (Batalex) [inline docs/how-to/h-integrate-alerts.md:9]: There should not be any difference between VM and k8s for the rest of the guide. The `grafana-agent` subordinate charm is mentioned in the Enable Monitoring guide

**Review by deusebio — CHANGES_REQUESTED** [review]
Great job! I liked the reading and it was actually simpler than expected (TWIL). I have one main suggestions which is to make it more like an how-to than a tutorial. Right now it feels a bit like a tutorial. There are few things we can probably do:

1. Use placeholder instead of concrete names
2. Move the examples to a separate place (I'm suggesting to create one branch in our kafka-test-app charm) and first state what they need to do, THEN provide them examples if they are interested to know more. 
3. Change the structure from a storyline and a specific use-case, to just provide the user what they need to custome their alerting rules/dashboard, in a concise manner. To this end, I believe it makes it more a how-to to rework slightly the structure, like:

    * Create Repo
    * Deploy cos-configuration-k8s (in a general way, without specifying either alert rules or dahsboards paths)
    * Configure and Relate:
        * alert rules
        * dashboards
        (alert rules and dashboards should feel like two non-mutually exclusive but optional items, with the same dignity)

File: docs/how-to/h-integrate-alerts.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+If your configuration differs, you will have to adapt this guide's content to your deployment.
+
+## Create a repo with alert rules
+
+To keep things simple, we will illustrate this guide with a single alert rule that Prometheus will fire if our Kafka cluster does not achieve high availability.
```
[the remark this answers is not in the document]
Comment (Batalex) [reply docs/how-to/h-integrate-alerts.md:14]: PR created: https://github.com/canonical/kafka-test-app/pull/16

[the remark this answers is not in the document]
Comment (Batalex) [reply docs/how-to/h-integrate-alerts.md]: With those suggestions, the guide is more concise and to-the-point, thank you.

**Review by deusebio — APPROVED** [review]
Looks good! When also @zmraul is happy, we can just update Discourse, and let Gatekeeper to raise the PR, closing this one referencing the automatically created one

**Review by zmraul — APPROVED** [review]
Very nice! TIL :)

**Comment by Batalex** [comment]
Thank you both for your review, the docs are live for both VM and K8s!

**Pull request closed without merging**